core/sql/ustat/hs_globals.h

```diff
@@ -1889,6 +1897,8 @@ class HSGlobalsClass : public NABasicObject
     Int64 stmtStartTime;
     NABoolean jitLogOn;
 
+    static THREAD_P NABoolean performISForMC_;
```

Comment:
I'm wondering why this is a static variable. The constructor always initializes it to FALSE, so if there is more than one HSGlobalsClass object created by a thread, one of the constructor calls will trash any value that was set by method calls on the other object. Perhaps only one HSGlobalsClass object is created per thread, but if that is the case, again I wonder why it is a static variable.